.github/workflows/build-and-install.yml

```diff
@@ -25,56 +26,92 @@ jobs:
           - 'fedora:32'
           - 'fedora:31'
           - 'fedora:30'
-            #- 'opensuse/leap:15.2'
-            #- 'opensuse/leap:15.1'
-            #- 'opensuse/tumbleweed:latest'
+          - 'opensuse/leap:15.2'
+          - 'opensuse/leap:15.1'
+          - 'opensuse/tumbleweed:latest'
```

Comment:
Was this on purrpose?

Reply:
Yes.

With `fail-fast: false` and the fact that I've turned off OpenSUSE as a required check, running these won't block PR's anymore. I'd suggest we reenable the Leap checks as they work fine and just leave Tumbleweed out of the required checks going forwards.